setup

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# -e (-o errexit): exit immediately on error
+# -u (-o nounset): treat unset variables as erros
+# -o pipefail: propagate intermediate errors in pipelines
+set -euo pipefail
+
+main() {
+  print_step "Verifying that pip is available"
+  pip=$(command -v pip3 || command -v pip || echo -n "")
+  if [ -z "$pip" ]; then
+    print_error "Cannot find pip, aborting."
+    exit 1
+  fi
+
+  print_step "Installing and initializing pipenv"
+  $pip install pipenv
+
+  print_step "Installing Python dependencies"
+  pipenv install
+
+  print_step "Generating Ansible Vault secret"
+  pwgen -1 24|tee ~/.forem_selfhost_ansible_vault_password
+
+  print_step "Copying example inventory"
+  cp -v inventory/example/setup.yml inventory/forem/setup.yml
+
+  print_step "Generating Vault secrets"
```

Comment:
I almost have a `sed` expression ready that will automatically add these to `inventory/forem/setup.yml` but it needs a bit more tweaking to deal better with YAML's whitespace-sensitive nature.